doc/articles/contributing/guidelines/breaking-changes.md

```diff
@@ -1,24 +1,45 @@
 # Guidelines for breaking changes
 
-## overview
+## Overview
 
 Uno uses a [Package Diff tool](https://github.com/unoplatform/uno.PackageDiff) to ensure that [binary breaking changes](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/breaking-changes#binary-breaking-change) do not
 go unnoticed. As [part of our continuous integration process](https://github.com/unoplatform/uno/blob/1a786a652394f5a3d674fadfdd7b459f8f476a1b/build/Uno.UI.Build.csproj#L201) the PackageDiffTool consumes the last published non-experimental package available on nuget.org, and compares it with the current PR.
 
 This process only diffs against previous versions of Uno, not against the UWP assemblies, so it doesn't pick up all forms of mismatches. There are [some inconsistencies](https://github.com/unoplatform/uno/pull/1300) dating from before SyncGenerator was added. At some point it might be a good idea to extend SyncGenerator tool to try to report them all (or even automatically fix them)
 
-## rule of thumb
+Breaking changes must be marked as such when committed using the [Conventional Commits formatting](../../uno-development/git-conventional-commits.md).
 
-* Binary breaking changes, by default, are considered unacceptable.
-* Uno is an implementation of the `Windows.UI.Xaml` APIs and as such the public API surface of UWP dictates the direction of Uno.
-* Thus, the rule of thumb is to correct Uno to match UWP but proceed with extra caution.
+## When are binary breaking changes acceptable?
 
-## blessing breaking changes
+### Breaking cross-platform compatibility - not ok
 
-In most cases, breaking changes are not acceptable, but in cases where there is no easy work around the [build/PackageDiffIgnore.xml](https://github.com/unoplatform/uno/blob/master/build/PackageDiffIgnore.xml) file can be adjusted to bless the changes. Please refer
-to the documentation of the [Uno.PackageDiff tool](https://github.com/unoplatform/uno.PackageDiff) for more information.
+Changes that break compatibility with the public API surface of UWP are generally never acceptable, because they not only break existing code but break cross-platform compatibility as well.
 
-## example report
+### Restoring cross-platform compatibility - ok, but discuss with core team
+
+In some cases, Uno's existing API is close to UWP, but not identical. (Hypothetical xample: a property with type `DependencyObject[]` on Uno, but type `IList<DependencyObject>` on UWP.) This is mostly the case for older code that was written before the use of [generated `NotImplemented` stubs](../../uno-development/uno-internals-overview.md#generated-notimplemented-stubs) and the `PackageDiff` tool, which act in combination to prevent these kinds of errors when implementing new features.
```

Comment:
```suggestion
In some cases, Uno's existing API is close to UWP, but not identical. (Hypothetical example: a property with type `DependencyObject[]` on Uno, but type `IList<DependencyObject>` on UWP.) This is mostly the case for older code that was written before the use of [generated `NotImplemented` stubs](../../uno-development/uno-internals-overview.md#generated-notimplemented-stubs) and the `PackageDiff` tool, which act in combination to prevent these kinds of errors when implementing new features.
```